code/write_excel/03_Write_Excel_4.R

```diff
@@ -106,7 +103,8 @@ kpi_3_2_res <- kpi_3_2_res |>
 
 ## KPI 3.2 HB of Surgery ----
 kpi_3_2_surg <- theme4_3 |> 
-  filter(kpi %in% c("KPI 3.2 Surgery")) |> 
+  filter(kpi %in% c("KPI 3.2 Surgery"),
+         !hbres == "Dumfries & Galloway") |> 
```

Comment:
I think we will actually need to add something in, either here or in the quarterly extracts check, to ask them to review this field. Let's discuss next year.